<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="IClabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ionisation Chamber Current</string>
    </property>
   </widget>
   <widget class="QLabel" name="Flux_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Flux Density</string>
    </property>
   </widget>
   <widget class="QLabel" name="KERMA_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>air KERMA rate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Current">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Flux">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_KERMA">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_I">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>uA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_F">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ph/mm^2/sec</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_D">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gy/sec</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Beam area set to (mm^2):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Beam energy set to (keV):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ionisation chamber energy loss correction:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Area">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Area</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Energy">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Energy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ke">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ke</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Current</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_Current()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_Flux</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_Flux()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_KERMA</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot_KERMA()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_KERMA()</slot>
  <slot>slot_Current()</slot>
  <slot>slot_Flux()</slot>
 </slots>
</ui>
